features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -436,7 +437,14 @@ class MessageComposerPresenter @Inject constructor(
                 val eventId = capturedMode.eventId
                 val transactionId = capturedMode.transactionId
                 timelineController.invokeOnCurrentTimeline {
+                    // First try to edit the message in the current timeline
                     editMessage(eventId, transactionId, message.markdown, message.html, message.mentions)
+                        .onFailure { cause ->
```

Comment:
Shouldn't this be `mapFailure` instead, so the result of `room.editMessage` is returned instead?

Reply:
U're right, good catch